src/CMakeLists.txt

```diff
@@ -316,11 +330,64 @@ foreach(_flag ${HPX_TARGET_COMPILE_DEFINITIONS_VAR})
   endif()
 endforeach()
 
+################################################################################
+# libhpx_wrap
+
```

Comment:
All of this should be disabled for anything but Linux

Reply:
oh yes! forgot that.